plugins/woocommerce/client/legacy/css/admin.scss

```
@@ -6451,10 +6451,10 @@ img.ui-datepicker-trigger {
 
 				&:hover {
 					box-shadow: inset 0 -1px 0 0 #dfdfdf,
-						inset 300px 0 0 rgba(156, 93, 144, 0.1);
-					border-right: 5px solid #9c5d90 !important;
+						inset 300px 0 0 #f7edf7;
```

The `#f7edf7` value here is the same as https://github.com/Automattic/color-studio/blob/4d85dab2331da9d6cf4ab7c42f1c208fc0d9efce/dist/color-variables.scss#L160.

We should use the defined variables instead of hardcoded values whenever possible. Here's an example below (I have not tested this): 

```suggestion
						inset 300px 0 0 $studio-woocommerce-purple-0;
```

You can search for more examples like this in the WooCommerce code base.

---

Thanks @ecgan for the suggestion. For this PR I have decided to skip this one. The variable `$studio-woocommerce-purple-0` is not accessible in this place because we do not have `@automattic/color-studio` installed as a dependency in the [woocommerce plugin](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/package.json) specifically. As the report section will go away eventually, I think can avoid adding an extra package for this feature.
cc: @mattallan

---

Okay sure, yeah let's do that. 🙂 👍